src/components/Instance/Definitions/index.svelte

```diff
@@ -0,0 +1,23 @@
+{#if definitions }
+  {#each definition_names as definition }
+   <dt>
+     <a on:click|preventDefault={select_instance(ref(definition))} href="#"
+       title={definition}>
+       {definition}</a>
+   </dt>
+  {/each}
+{/if}
+
+<script>
+  export let definitions = {}; //definitions section of the schema
+  export let select_instance;   //handler function 
+
+  let definition_names = [];
+  $: definition_names = Object.keys(definitions);
+
+  const ref = (str) => { return '#/definitions/'+str; };
```

Comment:
I know what you were asking earlier. Yeah, we'll need the url of the schema prepended to that so that the `select_instance` will know the full path to retrieve the definition.